ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterAggregator.java

```diff
@@ -19,17 +19,19 @@
 import com.github.ambry.utils.Pair;
 import java.io.IOException;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import org.codehaus.jackson.annotate.JsonAutoDetect;
 import org.codehaus.jackson.map.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 /**
  * Responsible for performing cluster wide stats aggregation.
  */
 public class HelixClusterAggregator {
+  private static final Logger LOGGER = LoggerFactory.getLogger(HelixClusterAggregator.class);
```

Comment:
why is this variable declared in caps? This is not how it is done elsewhere.
caps are used for integral constants and such. We don't do that for every final variable.